web/api/v1/api_remote_write.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
The date in the copyright is the date the file is created I think, so 2020.

Reply:
Ah right, that makes sense :)